Gemfile.lock

```diff
@@ -484,6 +484,7 @@ GEM
     sys-uname (1.2.3)
       ffi (~> 1.1)
     thor (1.2.2)
+    timeliness (0.4.5)
```

Comment:
@starswan From where this came?

Reply:
It was my first attempt - but it was overkill for the actual problem in the end. Just removed it with 'bundle install' - shame the build didn't fail somehow?